pkg/components/render_test.go

```diff
@@ -0,0 +1,72 @@
+package components
+
+import (
+	"bytes"
+	"testing"
+	"text/template"
+
+	"github.com/openshift/microshift/pkg/assets"
+	"github.com/openshift/microshift/pkg/config/lvmd"
+)
+
+func Test_renderLvmd(t *testing.T) {
+	expect := func(data []*lvmd.DeviceClass) []byte {
+		c := template.Must(template.New("").Parse(
+			`# Source: topolvm/templates/lvmd/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: lvmd
+  namespace: openshift-storage
+data:
+  lvmd.yaml: {{ .lvmd }}
+`))
```

Comment:
These tests look right, but I'd feel more comfortable with them if the expected values were spelled out by hand in the loop below, rather than using a duplicate template and rendering the same structures again in a different way.

Reply:
Fixed! :)